etcd/cluster-etcd-operator.md

```diff
@@ -0,0 +1,331 @@
+---
+title: cluster-etcd-operator
+authors:
+  - "@hexfusion"
+reviewers:
+  - "@abhinavdahiya"
+  - "@alaypatel07"
+  - "@crawford"
+  - "@deads2k"
+  - "@sttts"
+approvers:
+  - "@derekwaynecarr"
+  - "@smarterclayton"
+  - "@sttts"
+creation-date: 2019-10-1
+last-updated: 2019-10-2
+status: implementable
+see-also: 
+replaces:
+superseded-by:
+---
+
+# Cluster etcd Operator
+
+cluster-etcd-operator (CEO) is an operator that handles the scaling of etcd during cluster bootstrap and regular operation and provisioning etcd dependencies such as TLS certificates.
+
+## Release Signoff Checklist
+
+- [x] Enhancement is `implementable`
+- [ ] Design details are appropriately documented from clear requirements
+- [ ] Test plan is defined
+- [ ] Graduation criteria for dev preview, tech preview, GA
+- [ ] User-facing documentation is created in [openshift/docs]
+
+## Open Questions [optional]
+
+TODO
+
+## Summary
+
+cluster-etcd-operator (CEO) is an operator that handles the scaling of etcd during cluster bootstrap and regular operation and provisioning etcd dependencies such as TLS certificates.
+
+## Motivation
+
+Having an operator manage the etcd cluster members requires the ability to scale up and scale down the etcd membership ona Pod level independent of the Node itself. Meaning if you were to scale down from 5 to 4 that does not necessarily mean that you must remove the node from the cluster to achieve this. A good example would be replacing a failed member. In this case we need to remove the failed member (scale down to 4) and re-add (scale back to 5). This action would not require a new Node, only management of the static Pod on that Node. Scaling etcd requires a balance of proper configuration and timing to minimize stress to cluster.
```

Comment:
typo: `s/ona Pod/on a Pod/`

(I deleted the first comment I added, wrong line)